FlowCrypt/Controllers/Setup/SetupCreatePassphraseViewController.swift

```diff
@@ -23,14 +23,19 @@ enum CreateKeyError: Error {
     case conformingPassPhraseError
 }
 
+enum CreatePassphraseWithExistingKeyError: Error {
+    // No private key was found
+    case noPrivateKey
+}
+
 /**
  * Controller which is responsible for generating a new key during setup
  * - User is sent here from **SetupInitialViewController** in case there are no backups found
  * - Here user can enter a pass phrase (can be saved in memory or in encrypted storage) and generate a key
  * - After key is generated, user will be redirected to **main flow** (inbox view)
  */
 
-final class SetupGenerateKeyViewController: TableNodeViewController, PassPhraseSaveable {
+final class SetupCreatePassphraseViewController: TableNodeViewController, PassPhraseSaveable {
```

Comment:
Careful - here you are co-mingling two different setup flows into the same file, which I would like to avoid, for clarity. The way we have designed the setup flows is that each distinct flow gets a distinct VC. (you can still abstract the UI logic since it's similar - in both cases pass phrase needs to be created)

`SetupGenerateKeyViewController` was a situation when consumer-user installs the app, doesn't have any key, and is offered to create one and back it up in inbox. The name describes the intent well here.

I understand that the UI is very similar between that flow and what you are implementing, but the purpose is very different, and I'd like that to be clear from the code.

You could extract UI-specific stuff into a new abstract class `SetupCreatePassphraseAbstractViewController`, for example. Then you could update `SetupGenerateKeyViewController` to use it, but don't add EKM functionality there.

Then add a new class, `SetupWithKeysFromEkmViewController`, which will also inherit from newly created `SetupCreatePassphraseViewController`, and contain this newly added logic.

Then, finally, on the previous view, you forward user to `SetupWithKeysFromEkmViewController` when keys were fetched from EKM.

Then the distinction will be very clear.